src/current/v23.1/goldengate.md

```diff
@@ -129,10 +129,10 @@ This section describes how to configure Oracle GoldenGate for PostgreSQL to work
 [Extract](https://docs.oracle.com/goldengate/c1230/gg-winux/GGCON/processes-and-terminology.html) is Oracle GoldenGate's data capture mechanism that is configured to run against the source database.
 
 {{site.data.alerts.callout_success}}
-The steps in this section should be run on a machine and in a directory where Oracle GoldenGate for Oracle is installed.
+Complete the steps in this section on a machine and in a directory where Oracle GoldenGate for Oracle is installed.
```

Comment:
I think you might have an extra "for Oracle" here.

Reply:
unfortunately i think that is the correct product name, as awkward as it sounds. the other installation is called Oracle GoldenGate for PostgreSQL.